setup.cfg

```diff
@@ -0,0 +1,2 @@
+[flake8]
+extend-ignore = E203
```

Comment:
The gepetto linter config has linelength=119 and exclude cmake/, which this file would omit. Should we add it here as well?

Reply:
Good catch, i'll add this extend ignore to gepetto linters. But the linelength will have to move to 88 for black :/

Reply:
PS: excluding cmake is not mandatory anymore for flake, since https://github.com/jrl-umi3218/jrl-cmakemodules/pull/524